vec120/src/main/java/com/foursoft/vecmodel/vec120/HasOccurrenceOrUsages.java

```diff
@@ -0,0 +1,33 @@
+/*-
+ * ========================LICENSE_START=================================
+ * vec-common
+ * %%
+ * Copyright (C) 2020 - 2022 4Soft GmbH
+ * %%
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ * =========================LICENSE_END==================================
+ */
+package com.foursoft.vecmodel.vec120;
+
+import java.util.List;
+
+public interface HasOccurrenceOrUsages {
+
+    List<VecOccurrenceOrUsage> getOccurrenceOrUsage();
```

Comment:
Shouldn't that be `getOccurrenceOrUsages` (plural)?

Reply:
not possible the method in the vec model is called like this

Reply:
Okay, that's bad but explains it. Perhaps documentation should be added to reflect that, else you could assume that list is supposed to only contain one element.

Will that get fixed with VEC 2.0.0?

Reply:
Documation is generated via jaxb